sdk/python/examples/asyncpg_connect.py

```diff
@@ -1,10 +1,17 @@
 import asyncio
+from os import environ
 
-from approzium import Authenticator
+from approzium import AuthClient
 from approzium.asyncpg import connect
 from approzium.asyncpg.pool import create_pool
 
-auth = Authenticator("authenticator:6001")
```

Comment:
I like the more minimal examples better, since they show the essence. I suggest we make examples TLS-off by default, and have a couple or so that use TLS.

Reply:
I like that idea from a user experience perspective, but having TLS off by default would violate our secure-by-default architecture statement and we'd have to remove it. It is also standard for security products to have TLS on by default, [here's one example](https://www.vaultproject.io/docs/configuration/listener/tcp#tls_disable), [Thycotic's docs](https://thycotic.force.com/support/s/article/Issue-with-Disabling-TLS-1-0) indicate that TLS is on by default and heavily recommended, and [CyberArk recommends it as well](https://docs.cyberark.com/Product-Doc/OnlineHelp/PAS/Latest/en/Content/Security/Security%20Fundamentals-Introduction.htm) under their Use Secure Protocols section.

What would you think about leaving TLS on by default, and then give a mixture of examples so people have both in their toolbox?

Reply:
I am all for TLS on by default. But I was suggesting for the examples, use `disable_tls=True` for most, and add one or some that use it. That way, the examples would be shorter in general. My thinking is that shorter examples help put more emphasis on the actual feature that we want to show in that example and reduces duplicate code (for example, if we have an example for opentelemetry integration, showing how to use SSL is not important)

Reply:
Oh gotcha! Yes, definitely can do that! Thanks!

Reply:
I updated these in https://github.com/cyralinc/approzium/pull/111/commits/5a949ac2f0edb8885e7484703265634e369ba002, hopefully that is what you meant.